sdk-extension/opentelemetry-sdk-extension-aws/setup.cfg

```diff
@@ -23,7 +23,6 @@ url = https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/s
 platforms = any
 license = Apache-2.0
 classifiers =
-    Development Status :: 4 - Beta
```

Comment:
Should add the [stable](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/setup.cfg#L26) classifier.

Reply:
Thanks for that!